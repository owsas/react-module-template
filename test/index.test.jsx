import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Test from '../src/index';

describe('Test', () => {
  test('should be a component', () => {
    expect(<Test />).toBeTruthy();
  });

  test('should say "Hello, world!"', () => {
    const s = shallow(<Test />);
    expect(s.text()).toEqual('Hello world!');
  });

  test('should match snapshot', () => {
    const s = shallow(<Test />);
    expect(toJSON(s)).toMatchSnapshot();
  });
});
