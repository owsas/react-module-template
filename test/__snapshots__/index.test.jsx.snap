// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test should match snapshot 1`] = `
<div>
  Hello world!
</div>
`;
